### PR TITLE
Removes cultist space bases

### PR DESCRIPTION
**Pull request by Iamgoofball**
im the hatchet man

i eat garbage code and output fixed code people dislike

:cl: the hatchet man (i eat garbage code)
fix: you shitlords picked the wrong virtual anime tits to fuck so im takin away your space bases
experimental: lets be fair all you guys did in those was jack off to hentai, and not real man's gachimuchi
/:cl:

### CODE REVIEW

**Comment by kevinz000** [comment]
lol

**Comment by MrStonedOne** [comment]
>im the hatchet man

https://www.youtube.com/watch?v=fe8jJBoEmuY

**Review by ShizCalev — APPROVED** [review]

File: code/game/gamemodes/cult/ritual.dm [hunk 1]
```diff
@@ -269,5 +269,11 @@ This file contains the arcane tome files.
 	if(!is_station_level(T.z) && !is_mining_level(T.z))
 		to_chat(user, "<span class='warning'>The veil is not weak enough here.</span>")
 		return FALSE
+		
+	var/area/A = get_area(T)
+	if(A && !A.blob_allowed)
+		to_chat(user, "<span class='warning'>There's a passage in [src] specifically forbidding oyster consumption, triple-frying, and building outside of designated cult zones.</span>")
```
Comment (DaedalusGame) [inline code/game/gamemodes/cult/ritual.dm]: Hey I'm sorry but you have to code cultists being unable to triple-fry stuff too

Comment (MoreRobustThanYou) [inline code/game/gamemodes/cult/ritual.dm]: this is a masterpiece

**Pull request merged**